doc/source/rest.j2

```diff
@@ -187,6 +187,13 @@ can be resampled to a new |granularity|.
 
 {{ scenarios['get-measures-resample']['doc'] }}
 
+
+.. note::
+
+   If the quey is planned to be done often, for better performance, it is
```

Comment:
If you plan to execute the query often, it is recommended for performances to leverage an |archive policy| with the needed |granularity| instead of resampling the time series on each query.